minecraft_servers/justchunks/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "server_name": "justchunks",
+  "nice_name": "JustChunks",
+  "direct_ip": "mc.justchunks.net",
+  "server_wildcards": [
+    "%.justchunks.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en",
+    "es"
+  ],
+  "social": {
+    "web": "https://justchunks.net",
+    "web_shop": "https://shop.justchunks.net",
+    "web_support": "https://forum.timolia.de",
+    "twitter": "JustChunks",
+    "discord": "https://discord.gg/wf94dAA",
+    "tiktok": "justchunksnet",
+    "facebook": "JustChunks",
+    "youtube": "https://www.youtube.com/JustChunks",
+    "instagram": "JustChunks"
+  },
+  "gamemodes": {
+    "dungeon_explorer": {
+      "name": "Dungeon Explorer",
+      "color": "#ff5f14",
+      "url": "https://justchunks.net/game/dungeon-explorer",
+      "command": "/quickjoin dungeon",
+      "versions": "1.19.3<*"
+    },
+    "clashlands": {
+      "name": "Clashlands",
+      "color": "#d03b06",
+      "url": "https://justchunks.net/game/clashlands",
+      "command": "/quickjoin pvp",
+      "versions": "1.19.3<*"
+    }
+  },
+  "brand": {
+    "primary": "#24be62",
+    "background": "#24be62",
+    "text": "#ffffff"
+  },
+  "location": {
+    "city": "Falkenstein",
+    "country": "Germany",
+    "country_code": "DE"
+  },
+  "yt_trailer": "Ngi6n0XI2dM",
+  "user_stats": "https://justchunks.net/user/{userName}",
+  "discord": {
+    "server_id": 140450742329671680,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

Reply:
Ah, thanks for the info. I've changed it to `false` :+1: